lib/vagrant-service-manager/services/open_shift.rb

```diff
@@ -60,6 +63,30 @@ def self.print_info(ui, options)
         end
       end
 
+      def self.install_cli(machine, ui, options)
+        Installer.new('openshift', machine, ui, options).run
+      end
+
+      def self.binary_path(version)
+        "#{BIN_FOLDER}/openshift/#{version}/oc"
+      end
+
+      def self.binary_download_url(machine, version)
+        arch = machine.env.host.capability(:os_arch)
```

Comment:
Is this not exactly what we wanted to avoid? Now this service class knows how to download for ADB. What's about the CDK? Was the idea not to encapsulate this?